internal/action/otp.go

```diff
@@ -65,25 +66,52 @@ func (s *Action) otp(ctx context.Context, name, qrf string, clip, pw, recurse bo
 		skip = true
 	} else { // if not then we want to print a progress bar with the expiry time
 		out.Printf(ctx, "%s", token)
-		out.Warningf(ctx, "This OTP password still lasts for:", nil)
+		out.Warningf(ctx, "([q] to stop. -o flag to avoid.) This OTP password still lasts for:", nil)
 		bar := termio.NewProgressBar(int64(secondsLeft))
 		bar.Hidden = ctxutil.IsHidden(ctx)
 		if bar.Hidden {
 			skip = true
 		} else {
+			cancel := make(chan bool)
```

Comment:
I wonder if I'm missing something, but can't you just use `ctx, cancel := context.WithCancel(ctx)` and then use the standard [cancel](https://github.com/gopasspw/gopass/blob/master/pkg/hibp/dump/scanner.go#L175) idiom there?